### PR TITLE
Document upcoming media group plugin

### PR DESCRIPTION
**Pull request by KnorpelSenf**
https://github.com/grammyjs/grammY/issues/101

### CODE REVIEW

**Review by roj1512 — COMMENTED** [review]
Rename to `media-groups.md`?

**Comment by KnorpelSenf** [comment]
It's session and keyboard, not sessions and keyboards

File: site/docs/plugins/media-group.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Media Groups (built-in)
```
Comment (roj1512) [inline site/docs/plugins/media-group.md:1]: Why is it Media Groups here but the filename is media-group.md?

Comment (roj1512) [reply site/docs/plugins/media-group.md:1]: It is also mentioned as “media group” in the first sentence.

File: site/docs/plugins/media-group.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+const video = InputMediaBuilder.video(new InputFile("/tmp/video.mp4"));
+// etc
+```
+
+:::
+::::
+
+Check out all methods of `InputMediaBuilder` in the [API reference](https://deno.land/x/grammy/mod.ts?s=InputMediaBuilder).
+
+You can also directly pass URLs which Telegram fetches.
```
Comment (roj1512) [inline site/docs/plugins/media-group.md:50]:
```suggestion
You can also directly pass public URLs which Telegram fetches.
```

File: site/docs/plugins/media-group.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+```ts
+const photo InputMediaBuilder.photo("https://grammy.dev/images/Y.png", {
+  caption: "grammY is awesome",
+  // etc
+});
+```
+
+## Sending a Media Group
+
+You can send a media group as follows.
```
Comment (roj1512) [inline site/docs/plugins/media-group.md:67]:
```suggestion
You can send a media group as follows:
```

File: site/docs/plugins/media-group.md (hunk 1 above)
Comment (KnorpelSenf) [reply site/docs/plugins/media-group.md:1]: That's the terminology built-in plugins use. The session plugin deals with sessions. The keyboard plugin deals with keyboards. The media group plugin deals with media groups.

Comment (roj1512) [reply site/docs/plugins/media-group.md:1]: Thanks. I just figured that out by reading the docs.

**Review by roj1512 — APPROVED** [review]

File: site/docs/plugins/media-group.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+You can also directly pass public URLs which Telegram fetches.
+
+```ts
+const photo = InputMediaBuilder.photo("https://grammy.dev/images/Y.png");
+```
+
+Further options can be provided in an options object at the end.
+
+```ts
+const photo InputMediaBuilder.photo("https://grammy.dev/images/Y.png", {
```
Comment (Nazar-Ant) [inline site/docs/plugins/media-group.md:59]:
```suggestion
const photo = InputMediaBuilder.photo("https://grammy.dev/images/Y.png", {
```

File: site/docs/plugins/media-group.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+## Editing a Media Message
+
+Since `InputMedia` objects are also used to edit media messages, this plugin will assis you here, too:
+
+```ts
+const newMedia = InputMediaBuilder.photo("https://grammy.dev/images/Y.png");
+await ctx.editMessageMedia(newMedia);
+```
+
+As always, this works for `ctx.api.editMessageMedia` and `bot.api.editMessageMedia`, too.
```
Comment (Nazar-Ant) [inline site/docs/plugins/media-group.md:84]: Why is the usual plugin summary missing?

```suggestion
As always, this works for `ctx.api.editMessageMedia` and `bot.api.editMessageMedia`, too.

## Plugin Summary

This plugin is built-in into the core of grammY.
You don't need to install anything to use it.
Simply import everything from grammY itself.

Also, both the documentation and the API reference of this plugin are unified with the core package.
```

Comment (roj1512) [reply site/docs/plugins/media-group.md:84]: It’s been a while since Steffen has written docs.

Comment (roj1512) [reply site/docs/plugins/media-group.md:84]: And it’s been a while since I have reviewed with a really open eye and out-of-the-box brain like yours.

Comment (KnorpelSenf) [reply site/docs/plugins/media-group.md:84]: ffs

**Review by roj1512 — APPROVED** [review]

**Review by CikiMomogi — APPROVED** [review]

File: site/docs/id/plugins/media-group.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+Kamu bisa menggunakan plugin dengan cara seperti ini:
+
+::::code-group
+:::code-group-item TypeScript
+
+```ts
+import { InputMediaBuilder } from "grammy";
+
+const photo = InputMediaBuilder.photo(new InputFile("/tmp/foto.mp4"));
```
Comment (fadzikri) [inline site/docs/id/plugins/media-group.md:18]: do we need to translate variable too?

File: site/docs/.vuepress/configs/id.ts [hunk 7]
```diff
@@ -166,6 +166,10 @@ export const localeId: LocaleConfig<DefaultThemeLocaleData> = {
                 text: "Keyboard Custom dan Inline",
                 link: "/id/plugins/keyboard.html",
               },
+              {
+                text: "Media Groups",
```
Comment (fadzikri) [inline site/docs/.vuepress/configs/id.ts:170]:
```suggestion
                text: "Pengelompokan Media",
```

File: site/docs/id/plugins/media-group.md (hunk 6 above)
Comment (CikiMomogi) [reply site/docs/id/plugins/media-group.md:18]: I think we can leave it as it is.

Comment (fadzikri) [reply site/docs/id/plugins/media-group.md:18]: okay

**Review by CikiMomogi — APPROVED** [review]

**Review by fadzikri — APPROVED** [review]

File: site/docs/uk/plugins/media-group.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
+# Медіагрупи (вбудовано)
+
+Плагін медіагрупи допоможе вам надсилати групи мультимедійних даних, дозволяючи створювати обʼєкти `InputMedia`.
```
Comment (LWJerri) [inline site/docs/uk/plugins/media-group.md:3]:
```suggestion
Плагін медіагруп допоможе вам надсилати групи мультимедійних даних, дозволяючи створювати обʼєкти `InputMedia`.
```

**Review by LWJerri — APPROVED** [review]

File: site/docs/uk/plugins/media-group.md (hunk 8 above)
Comment (Nazar-Ant) [reply site/docs/uk/plugins/media-group.md:3]: До цього ми використовували назву плагіну в однині https://github.com/grammyjs/website/blob/c229f7a2f640fd600e3dc2c82fdd182582c488e8/site/docs/uk/plugins/session.md?plain=1#L60

**Review by habemuscode — APPROVED** [review]

File: site/docs/.vuepress/configs/es.ts [hunk 9]
```diff
@@ -164,6 +164,10 @@ export const localeEs: LocaleConfig<DefaultThemeLocaleData> = {
                 text: "Teclados lineales y personalizados",
                 link: "/es/plugins/keyboard.html",
               },
+              {
+                text: "Medios de comunicación",
+                link: "/plugins/media-group.html",
```
Comment (CikiMomogi) [inline site/docs/.vuepress/configs/es.ts:169]:
```suggestion
                link: "/es/plugins/media-group.html",
```

**Review by CikiMomogi — CHANGES_REQUESTED** [review]

**Review by CikiMomogi — APPROVED** [review]

File: site/docs/uk/plugins/media-group.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+## Редагування медіаповідомлень
+
+Оскільки обʼєкти `InputMedia` також використовуються для редагування медіаповідомлень, цей плагін допоможе вам і тут:
+
+```ts
+const newMedia = InputMediaBuilder.photo("https://grammy.dev/images/Y.png");
+await ctx.editMessageMedia(newMedia);
+```
+
+Як і завжди, це працює для `ctx.api.editMessageMedia` та `bot.api.editMessageMedia`.
```
Comment (deptyped) [inline site/docs/uk/plugins/media-group.md:84]:
```suggestion
Як і завжди, це також працює для `ctx.api.editMessageMedia` та `bot.api.editMessageMedia`.
```

**Comment by roj1512** [comment]
Wrong commit message but who cares

**Pull request merged**